app/policies/fund_policy.rb

```diff
@@ -1,22 +1,22 @@
 class FundPolicy < ApplicationPolicy
   def index?
-    true
+    user.administrator? || user.organisations.include?(@record.organisation)
   end
 
   def show?
-    true
+    user.administrator? || user.organisations.include?(@record.organisation)
   end
 
   def create?
-    true
+    user.administrator?
```

Comment:
We should also allow users who belong to that organisation to create funds. We can do this by setting the association in the method before calling `authorize`. Same for update and destroy.